### PR TITLE
feat: adding fast forward api

### PR DESCRIPTION
**Pull request by shariffdev**
PR adds the fastForward api and viewBlock api

### CODE REVIEW

**Comment by shariffdev** [comment]
@ailisp Ready to review

**Comment by shariffdev** [comment]
@fospring ready to review

**Comment by fospring** [comment]
> @fospring ready to review

LGTM

File: packages/js/src/jsonrpc.ts [hunk 1]
```diff
@@ -183,6 +183,23 @@ export class JsonRpcProvider extends JSONRpc {
...
     return this.sendJsonRpc('sandbox_patch_state', records);
   }
+
+  /**
+   * Fast forward to a point in the future. The delta block height is supplied to tell the
+   * network to advanced a certain amount of blocks. This comes with the advantage only having
+   * to wait a fraction of the time it takes to produce the same number of blocks.
+   *
+   * Estimate as to how long it takes: if our delta_height crosses `X` epochs, then it would
+   * roughly take `X * 5` seconds for the fast forward request to be processed.
```
Comment (gagdiez) [inline packages/js/src/jsonrpc.ts:193]:
```suggestion
   * roughly take `X * 5` miliseconds for the fast forward request to be processed.
```

**Review by gagdiez — APPROVED** [review]

**Pull request merged**